jazzy.sh

```diff
@@ -22,7 +22,10 @@ if [ -z "${GITHUB_USERNAME}" ] && [ -z "${GITHUB_PASSWORD}" ]; then
 fi
 
 # Check if .jazzy.yaml exists in the root folder of the repo
-if [ -e "${projectFolder}"/.jazzy.yaml ]; then
+#cd ..
+export projectFolder=`pwd`
+
+if [ -e ./$(projectFolder)/.jazzy.yaml ] || [ -e $(projectFolder)/tests/library/.jazzy.yaml ]; then
```

Comment:
Why do we need a `.jazzy.yaml` in the tests?

Reply:
This was a previous test, has now been removed.